graaljs/graaljs-gradle-webpack-guide/src/main/java/org/example/App.java

```diff
@@ -0,0 +1,44 @@
+package org.example;
+
+
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.HostAccess;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import org.graalvm.polyglot.proxy.ProxyExecutable;
+
+public class App {
+    public App() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        try (Context context = Context.newBuilder(new String[]{"js"})
```

Comment:
Can just be:
```suggestion
        try (Context context = Context.newBuilder("js")
```

In fact, you could just copy the files from the Maven guide, for example https://github.com/graalvm/graal-languages-demos/blob/0951909cb64a04262eaa6a27bc5800846e785019/graaljs/graaljs-maven-webpack-guide/src/main/java/com/example/App.java